manifests/nodeExporter-prometheusRule.yaml

```
@@ -134,6 +134,28 @@ spec:
       for: 1h
       labels:
         severity: critical
+
+    - alert: NodesMemoryFillingUp
+      annotations:
+        description: Average nodes memory utilization is {{ printf "%.2f" $value }}% and is filling up.
+        runbook_url: https://runbooks.prometheus-operator.dev/runbooks/node/nodesmemoryfillingup
+        summary: Average nodes memory utilization is high.
+      expr: |
+        ((Sum(node_memory_MemTotal_bytes) - Sum(node_memory_MemAvailable_bytes)) / Sum(node_memory_MemAvailable_bytes) * 100) > 80
```

Not sure if the expression is 100% correct, shouldn't it be
```suggestion
       (1 - (Sum(node_memory_MemAvailable_bytes) / Sum(node_memory_MemTotal_bytes))) * 100) > 80
```
We could also just flip it and remove the subtraction and say that we only have x% of memory left

---

Should not make a difference . But, memory utilization seems more urgent than memory left IMO.